python/ray/autoscaler/v2/instance_manager/cloud_providers/kuberay/cloud_provider.py

```diff
@@ -422,14 +432,19 @@ def _get_workers_groups_with_deletes(
             worker_groups_with_deletes.add(node_type)
 
             for worker in workersToDelete:
+                worker_to_delete_set.add(worker)
                 if worker in node_set:
                     worker_groups_with_pending_deletes.add(node_type)
                     break
 
         worker_groups_with_finished_deletes = (
             worker_groups_with_deletes - worker_groups_with_pending_deletes
         )
-        return worker_groups_with_pending_deletes, worker_groups_with_finished_deletes
+        return (
```

Comment:
nit: function signature as well.

Reply:
rename the function?

Reply:
renamed the function: https://github.com/ray-project/ray/pull/48909/commits/24ba62a6b198f1e2d2ee9ceb029e2bbb612d3849

Reply:
good catch! updated https://github.com/ray-project/ray/pull/48909/commits/2c9c3d0afe84f5f1c727b435960206399d573eb7